src/ui.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Container,
+  render,
+  Text,
+  VerticalSpace,Code,Space
+} from "@create-figma-plugin/ui";
+import { h } from "preact";
+import { useEffect } from "preact/hooks";
+import {copyToClipboard} from 'figx'
+
+function UI({ content }: { content: string} ) {
```

Comment:
**Suggestion**: Even if the library we use guarantees the received content is a string here, the reality is that it's `unknown` and that we should check it's a string before doing anything. This way we also separate the connected-to-external-world component from the shown one. It's an improvement, anyway, not a blocker at all 😊